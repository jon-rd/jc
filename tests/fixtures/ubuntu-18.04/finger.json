[{"login":"kbrazil","name":"Kelly Brazil","tty":"tty1","idle":"14d","login_time":"Mar 22 21:14","tty_writeable":false,"idle_minutes":0,"idle_hours":0,"idle_days":14,"total_idle_minutes":20160},{"login":"kbrazil","name":"Kelly Brazil","tty":"pts/0","idle":null,"login_time":"Apr  5 15:33","details":"(192.168.1.22)","tty_writeable":true,"idle_minutes":0,"idle_hours":0,"idle_days":0,"total_idle_minutes":0},{"login":"kbrazil","name":"Kelly Brazil","tty":"pts/1","idle":"1:00","login_time":"Apr  6 09:04","details":"(192.168.1.22)","tty_writeable":true,"idle_minutes":0,"idle_hours":1,"idle_days":0,"total_idle_minutes":60},{"login":"kbrazil","name":"Kelly Brazil","tty":"pts/0","idle":"14","login_time":"Apr  5 12:30","tty_writeable":true,"idle_minutes":14,"idle_hours":0,"idle_days":0,"total_idle_minutes":14},{"login":"kbrazil","name":"Kelly Brazil","tty":"pts/0","idle":"14","login_time":"Apr  5 2020","tty_writeable":true,"idle_minutes":14,"idle_hours":0,"idle_days":0,"total_idle_minutes":14},{"login":"kbrazil","name":"Kelly Brazil","tty":"pts/0","idle":"14","login_time":"Apr  5 2021","details":"(192.168.1.23)","tty_writeable":true,"idle_minutes":14,"idle_hours":0,"idle_days":0,"total_idle_minutes":14},{"login":"kbrazil","name":"Kelly Brazil","tty":"pts/1","idle":"1:00","login_time":"Apr  6 09:04","details":"(192.168.1.22)   other details","tty_writeable":true,"idle_minutes":0,"idle_hours":1,"idle_days":0,"total_idle_minutes":60}]
