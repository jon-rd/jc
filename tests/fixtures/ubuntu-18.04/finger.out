Login     Name           Tty      Idle  Login Time   Office     Office Phone
kbrazil   Kelly Brazil  *tty1      14d  Mar 22 21:14
kbrazil   Kelly Brazil   pts/0          Apr  5 15:33 (192.168.1.22)
kbrazil   Kelly Brazil   pts/1    1:00  Apr  6 09:04 (192.168.1.22)
kbrazil   Kelly Brazil   pts/0      14  Apr  5 12:30
kbrazil   Kelly Brazil   pts/0      14  Apr  5 2020
kbrazil   Kelly Brazil   pts/0      14  Apr  5 2021  (192.168.1.23)
kbrazil   Kelly Brazil   pts/1    1:00  Apr  6 09:04 (192.168.1.22)   other details
